index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> Building a Survey Form </title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+    <h1>Microverse Survey Form</h1>
+    <form id="servey-form">
+    <div class="form-control">
+        <label  id="name-label">Name: </label>
+        <input type="text" id="name" placeholder="Enter your name" />
+    </div>
+    <div class="form-control">
+        <label for="email-label">Email: </label>
+        <input type="email" id="email" placeholder="Enter your email" required>
+      </div>
+      
+        <div class="form-control">
+          <label for="number"> Age:</label> <input type="text" id="number-label" placeholder="Enter your age" required min="18" max="60" title="enter numeric values between 18 and 60" pattern="\d+"> 
+        </div>
+      
+        <div class="form-control">
+            <label  id="label-role">Which option best describes you? </label>
+    <br>
```

Comment:
Please make sure that all the lines of code are properly indented for easy reading of lines of code.

Reply:
- [ ] [OPTIONAL] Goog job :+1:  however I would like to recommend you use a couple of emojis and/or gifs in the review. in order to make the code review more human-friendly 👽🤖. :smile: